Sources/Locker/Core/Locker.swift

```diff
@@ -15,7 +15,8 @@ public class Locker: NSObject {
     // MARK: - Public properties
 
     /**
-     User defaults used for storing shouldUseAuthenticationWithBiometrics, askToUseAuthenticationWithBiometrics and shouldAddPasscodeToKeychainOnNextLogin values
+     User defaults used for storing shouldUseAuthenticationWithBiometrics,
+     askToUseAuthenticationWithBiometrics and shouldAddPasscodeToKeychainOnNextLogin values
```

Comment:
This should be a one-liner. In the documentation section, everything moved to 2+ line is parsed as "description", while only the first line is defined as a "short description".